var/spack/repos/builtin/packages/argobots/package.py

```diff
@@ -18,8 +18,10 @@ class Argobots(AutotoolsPackage):
     homepage = "http://www.argobots.org/"
     url      = "https://github.com/pmodels/argobots/releases/download/v1.0b1/argobots-1.0b1.tar.gz"
     git      = "https://github.com/pmodels/argobots.git"
+    maintainers = ['shintaro-iwasaki']
 
     version("develop", branch="master")
```

Comment:
You can actually rename this version of `master` now that #1983 has been merged

Reply:
Thanks! We changed the name from `develop` to `master`.